nixos/modules/services/matrix/mautrix-meta.nix

```
@@ -541,6 +541,7 @@ in {
 
             bridge = {
               username_template = mkDefault "facebook_{{.}}";
+              management_room_text.welcome = "Hello, I'm an Facebook bridge bot.";
```

And very similar mistakes are all over the diff..

```suggestion
              management_room_text.welcome = "Hello, I'm a Facebook bridge bot.";
```